packages/p2p-v2/src/pages/my-ads/screens/CreateEditAd/CreateEditAd.tsx

```diff
@@ -149,7 +161,7 @@ const CreateEditAd = () => {
             setValue('min-completion-rate', advertInfo.min_completion_rate);
             setValue('min-join-days', advertInfo.min_join_days);
             setValue('min-order', advertInfo.min_order_amount);
-            setValue('rate-value', advertInfo.rate);
+            setValue('rate-value', setInitialAdRate() as string);
```

Comment:
following above

```suggestion
            setValue('rate-value', setInitialAdRate());
```

Reply:
the typecasting was not done for handling the undefined. its for the type returned from the setinitialadrate